symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/ActivityRegistry.java

```diff
@@ -42,6 +46,11 @@ public void register(final AbstractActivity<?, ?> activity) {
   }
 
   private void preProcessActivity(AbstractActivity<?, ?> activity) {
+    // check if the activity is a help command or not
+    // If yes, replace the existing one
+    if (activity instanceof HelpCommand && this.activityList.size() > 0) {
+      this.activityList.set(0, activity);
```

Comment:
aren't we going to end up with two help command? (    this.activityList.add(activity); in register ())

this.activityList.size() > 0 is more expressive with !this.activityList.isEmpty() (i.e not empty vs size greater than 0)

Reply:
furthermore activityList is exposed (@getter) 
what if I mess up with the list, add my activity in position 0 and then add my custom help command?

Reply:
now to solve you might wonder if activity should have a sort of identity to be able to replace activities if we have duplicates
maybe this could be the activity info object? it could work for slash commands but might be limited for other types of commands

Reply:
I just found another issue is that if we mess up with the list, we will also have the conflict between the activity list and the datafeedloop subscribers. So I think we should only return a copy of the activity by the getter, right?